tests/codegen/issues/issue-114312.rs

```diff
@@ -0,0 +1,24 @@
+// compile-flags: -O
+// min-llvm-version: 16
```

Comment:
This isn't going to work with vanilla LLVM 16, but a "system" qualifier on 17 will still let it run with our bundled 16.
```suggestion
// min-system-llvm-version: 17
```

Reply:
I get it, thanks.